src/server/pbsd_init.c

```diff
@@ -275,19 +274,18 @@ init_server_attrs()
 		PBS_MIN_LICENSING_LICENSES;
 	server.sv_attr[(int)SRV_ATR_license_min].at_flags =
 		ATR_VFLAG_DEFLT|ATR_VFLAG_SET|ATR_VFLAG_MODCACHE;
+	licensing_control.licenses_min = PBS_MIN_LICENSING_LICENSES;
```

Comment:
Are they used for the same purpose? If so, you can remove one.

Reply:
We can keep this similarly we use the license location value as well as keep that in the server attribute as well